packaging/docker/run.sh

```diff
@@ -25,6 +25,7 @@ if [ -n "${NETDATA_CLAIM_URL}" ] && [ -n "${NETDATA_CLAIM_TOKEN}" ] && [ ! -f /v
                              -url "${NETDATA_CLAIM_URL}" \
                              ${NETDATA_CLAIM_ROOMS:+-rooms "${NETDATA_CLAIM_ROOMS}"} \
                              ${NETDATA_CLAIM_PROXY:+-proxy "${NETDATA_CLAIM_PROXY}"}
+                             -daemon-not-running
```

Comment:
1. That condition is not correct

https://github.com/Ferroin/netdata/blob/b8ff8fbed3a2c1e670af64a668552069b3071289/packaging/docker/run.sh#L23-L24

i dont know that is `/var/lib/netdata/claim.d/claimed_id`. Perhaps some old dir?

2. There is no `NETDATA_CLAIM_ROOMS` check. Is it ok to have `-rooms=`. Does it work?

Reply:
1. No, just me using the wrong name.
2. AIUI from discussions about how claiming works that I0ve had with a couple of people, we do not need to specify rooms. Not doing so claims the node, but does not add it to any of the rooms in the space, but it can be manually added later through Netdata Cloud.